config/src/test/scala/com/typesafe/config/impl/ConfigTest.scala

```diff
@@ -858,7 +858,7 @@ class ConfigTest extends TestUtils {
         if (home != null) {
             assertEquals(home, conf.getString("system.home"))
         } else {
-            assertEquals(nullValue, conf.getObject("system").get("home"))
+            assertEquals(null, conf.getObject("system").get("home"))
```

Comment:
glancing through this on my phone, it isn't obvious to me what this change is about?